.github/workflows/build.yml

```diff
@@ -10,7 +10,7 @@ on:
 jobs:
 
   tests:
-    runs-on: macos-latest
+    runs-on: macos-11
```

Comment:
It supposed to be same thing: [macos-11 = macos-latest](https://github.com/actions/virtual-environments/issues/4060)